docs/modules/ROOT/pages/cicd_concept.adoc

```diff
@@ -0,0 +1,30 @@
+= Continuous Delivery
+
+In order to ensure that code released to production is of the highest possible quality, we recommend
+setting up three separate deployment environments to play different roles in the Continuous Integration 
+/ Continuous Delivery process.
+
+== Target Enviroments
+
+A typical CI/CD framework would deploy images to the following three environments:
+
+=== Development
+
+The development environment is used for internal testing of infrastructure, such as endpoints,
+pipelines, back-end authentication, load tests, and database configuration. In order to be free
+to change or take down the development environment at will, we recommend against giving 
+external organizations any access to it.
```

Comment:
Is this really imperative? Are "external organizations" meant to be parties that are involved in the development process (e.g. QA, designers, etc.)?

We should maybe mention the use of "review apps" in the development environment, which makes "taking down" an action that is decoupled from an "environment" (as it is built afresh and taken down for every MR independently).